Composer/packages/extensions/adaptive-form/src/components/fields/ExpressionField/ExpressionEditor.tsx

```diff
@@ -25,7 +29,7 @@ const ExpressionEditor: React.FC<FieldProps> = (props) => {
         errorMessage: { display: 'none' },
       }}
       value={value}
-      onChange={handleChange}
+      onChange={onChange}
```

Comment:
I'm not sure you can get rid of the handleChange - it was sending the 2nd param as the value.  I think this would change the FormField<T>.onChange<T> to have the T be the event rather than the value.

Reply:
Actually no. The only thing that handleChange did is get the newValue and call onChange. The event was not used.